src/precice/impl/WatchPoint.cpp

```diff
@@ -93,35 +88,6 @@ void WatchPoint::initialize()
   }
 
   PRECICE_DEBUG("Rank: " << utils::MasterSlave::getRank() << ", isClosest: " << _isClosest);
```

Comment:
We could clear the `_interpolationElements` when `isClosest == false` this may prevent some mistakes in the future.